scripts/easy_train.py

```diff
@@ -1433,74 +1446,90 @@ def str2bool(v):
+    parser.add_argument("--nnue-pytorch-branch", type=str, default='glinscott/nnue-pytorch/master', dest='nnue_pytorch_branch', help="Path to the commit/branch to use for the trainer being tested.")
+    parser.add_argument("--build-engine-arch", type=str, default='x86-64-modern', dest='build_engine_arch', help="ARCH to use for engine compilation, e.g. x86-64-avx2 for recent hardware")
+    parser.add_argument("--build-threads", type=int, default=8, dest='build_threads', help="Number of threads to use for compilation")
     parser.add_argument("--fail-on-experiment-exists", type=str2bool, default=True, dest='fail_on_experiment_exists', help="By default an experiment must be created in an empty directory. Should only be used for debugging.")
     parser.add_argument("--epoch-size", type=int, default=100000000, dest='epoch_size', help="Number of positions per epoch.")
     parser.add_argument("--validation-size", type=int, default=1000000, dest='validation_size', help="Number of positions per validation step.")
     parser.add_argument("--tui", type=str2bool, default=True, dest='tui', help="Whether to show a nice TUI.")
     parser.add_argument("--do-network-testing", type=str2bool, default=True, dest='do_network_testing', help="Whether to test networks as they are generated.")
     parser.add_argument("--do-network-training", type=str2bool, default=True, dest='do_network_training', help="Whether to train networks.")
-    parser.add_argument("--network-testing-threads", type=int, default=1, dest='network_testing_threads', help="Number of threads to use for network testing.")
+    parser.add_argument("--network-testing-threads", type=int, default=default_testing_threads, dest='network_testing_threads', help="Number of threads to use for network testing. By default the available number of threads - minus data loader and pytorch threads. The optimal value might depend on the --threads, --num-workers and other machine load.")
     parser.add_argument("--network-testing-explore-factor", type=float, default=1.5, dest='network_testing_explore_factor', help="Elo error estimates are multiplied by this amount to determine testing candidates.")
-    parser.add_argument("--network-testing-book", type=str, default='./noob_3moves.epd', dest='network_testing_book', help="Path to a suitable book, see https://github.com/official-stockfish/books")
+    parser.add_argument("--network-testing-book", type=str, default='https://github.com/official-stockfish/books/raw/master/UHO_XXL_+0.90_+1.19.epd.zip', dest='network_testing_book', help="Path to a suitable book, or suitable link see https://github.com/official-stockfish/books")
     parser.add_argument("--network-testing-time-per-move", type=float, default=None, dest='network_testing_time_per_move', help="Number of seconds per game")
```

Comment:
`--network-testing-time-per-move` should indeed be named `--network-testing-time-per-game`

Reply:
done.